### ISSUE TITLE
Video playback windows is solid BLUE, no video output

### ISSUE THREAD

**Issue by RedKong** [jellyfin/jellyfin-desktop#200]
**Describe the bug**
Updated from an old MPV Shim version (don't remember which one, several months old) that was working flawlessly.
Now the video playback is not working with any files I have tried. It just shows a blue picture, I can press "C" to open the menu but the menu text is also invisible. The audio seems to be working fine.

I tried completely uninstalling and reinstalling, reinstalling with admin privilege, using the portable .exe, restarting the PC. 
No changes.

**To Reproduce**
1. Launch Jellyfin desktop
2. open any video files
3. blue window opens

**Expected behavior**
it should display the movie correctly,

**Screenshots**
![blue window](https://user-images.githubusercontent.com/52454506/113640570-0d689180-967c-11eb-823a-b2b7e6158de2.png)

**Desktop (please complete the following information):**
 - OS: Windows 20H2
 - Version v1.10.3 - Desktop Client Hotfix 3

**Error Messages**
[log.txt](https://github.com/jellyfin/jellyfin-desktop/files/6261159/log.txt)

I enabled logging via the config file (as the menu is not working.), relaunched the client, launched a movie, got the blue window, closed the client, uploaded the log.txt

**Comment by iwalton3** [comment]
Try pressing `k` while media is playing.

**Comment by RedKong** [comment]
> Try pressing `k` while media is playing.

Thanks for the reply.

I did that and now playback appears to be working correctly.

If I understand correctly "k" disables shader packs inside MPV?

**Comment by iwalton3** [comment]
Yes. So there must be some kind of driver issue on your computer. That feature just doesn't work for some people.

**Comment by RedKong** [comment]
Ok, if you wish to troubleshoot this further I am available.
I am curious about the driver issue, everything else seems to be working fine on this PC, from gaming to production.

Otherwise, I wasn't really using the shader feature, so this could be marked as a work-around for this problem. 
Thank you.

**Comment by iwalton3** [comment]
I'm not sure. It seems to be hardware specific.

**Comment by GuardToad** [comment]
I have the same issue but only for Anime4K X4 Perceptual (For SD), Anime4K X4 Perceptual + Deblur (For SD), Anime4K X2 Faithful (For HD), Anime4K X2 Perceptual (For HD) , Anime4K X2 Perceptual + Deblur (For HD). 
The screen turns green if u change video playback profile with something already playing and then blue when u close the player and play something. Changing gpu_api to vulcan in Jellyfin "Desktop\jellyfin_mpv_shim\default_shader_pack\pack" fixes the blue/green. 
But that also introduces a new issue where the player will open in fullscreen but is invisible until u press "Esc" then the player is viable but u get a black screen closing the player and opening it again makes it work as expected.
Not a perfect workaround by any means but if u really wanna use shaders and have the same issue this is what worked for me.

**Comment by jfml** [comment]
@GuardToad That works perfectly, thanks! It needs to be „vulkan“ with a k not a c, though, if I'm not completely mistaken (this confuses me so much because in German „Vulkan“ is actually written with a k and it's not just an edgy marketing stunt).

**Comment by GuardToad** [comment]
@jfml you're completely right. Typo on my part, glad it fixed it 😀
